### ISSUE TITLE
E tensorflow/core/grappler/clusters/utils.cc:87] Failed to get device properties, error code: 3

### ISSUE THREAD

**Issue by LeopoldACC** [tensorflow/tensorflow#48611]
### System information
-   **OS Platform and Distribution (e.g., Linux Ubuntu 18.04)**:
-   **TensorFlow installed from (source or binary)**:source
-   **TensorFlow version (use command below)**:1.15.4
-   **Keras version()**:2.3.1
-   **Python version**:3.6.3
-   **GCC/Compiler version (if compiling from source)**:7.5
-   **CUDA/cuDNN version**:1.11.0 / 8.0.4
-   **GPU model and memory**:RTX3090 24GB

### Describe the problem
the code I run no problem as I use RTX2070 with tensorflow1.13 keras2.2.4 and cuda10.0,but when I transfer my code to the new machine which has RTX3090,I tried to enable the oldest code and model,which lead to error when use `normalization` and **stop at the tf.TF_SessionRun_wrapper()** and can not dive into the deeper code.

- segmentation code traceback(by manual track)
1 
/home/dell/anaconda3/envs/dxjc/lib/python3.6/site-packages/efficientnet/model.py 
line 390 
**x = layers.BatchNormalization(axis=bn_axis, name='stem_bn')(x)**
2 
/home/dell/anaconda3/envs/dxjc/lib/python3.6/site-packages/keras/layers/convolutional.py
line 156
3 
/home/dell/anaconda3/envs/dxjc/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py
line 1858
4
/home/dell/anaconda3/envs/dxjc/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py
line 292
5
/home/dell/anaconda3/envs/dxjc/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py
line 278
6
/home/dell/anaconda3/envs/dxjc/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py
line 198
7
/home/dell/anaconda3/envs/dxjc/lib/python3.6/site-packages/tensorflow_core/python/client/session.py
line 955
line 1091
line 1318
line 1441 
**return tf_session.TF_SessionRun_wrapper(self._session, options, feed_dict,
                                            fetch_list, target_list,
                                            run_metadata)**

- detection traceback(by manual track)
1
/home/dell/anaconda3/envs/dx/lib/python3.6/site-packages/keras_applications/mobilenet_v2.py
**x = layers.BatchNormalization(axis=channel_axis,epsilon=1e-3,momentum=0.999,name='bn_Conv1')(x)**
2
/home/dell/anaconda3/envs/dx/lib/python3.6/site-packages/keras/engine/base_layer.py
line 451
3
/home/dell/anaconda3/envs/dx/lib/python3.6/site-packages/keras/layers/normalization.py 
line 183
normed_training, mean, variance = K.normalize_batch_in_training(
            inputs, self.gamma, self.beta, reduction_axes,
            epsilon=self.epsilon)
4
/home/dell/anaconda3/envs/dx/lib/python3.6/site-packages/keras/layers/normalization.py 
line 296
_LOCAL_DEVICES = get_session().list_devices()
line 503
5
/home/dell/anaconda3/envs/dx/lib/python3.6/site-packages/tensorflow_core/python/keras/backend.py
line 903
6
/home/dell/anaconda3/envs/dxjc/lib/python3.6/site-packages/tensorflow_core/python/client/session.py
line 955
line 1091
line 1318
line 1441 
**return tf_session.TF_SessionRun_wrapper(self._session, options, feed_dict,
                                            fetch_list, target_list,
                                            run_metadata)**

### Source code / logs
```python
E tensorflow/core/grappler/clusters/utils.cc:87] Failed to get device properties, error code: 3
```

**Comment by tilakrayal** [comment]
@LeopoldACC ,

TensorFlow 1.x is not actively supported. Could you please update TensorFlow to the latest stable version v2.4.1 and check if you are facing the same issue. Thanks!
